src/frame/js/smooch.jsx

```diff
@@ -0,0 +1,390 @@
+import React from 'react';
+import { render as reactRender, unmountComponentAtNode } from 'react-dom';
+import pick from 'lodash.pick';
+import { batchActions } from 'redux-batched-actions';
+import { Provider } from 'react-redux';
+
+import '../stylesheets/main.less';
```

Comment:
eheh, in the webpack config, less files are configured to use the style-loader which automatically inject the stylesheet in the page when required.